src/components/Home/styles.js

```diff
@@ -2,9 +2,26 @@ import { StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
   container: {
+    backgroundColor: 'black',
```

Comment:
You could maybe have a Colors file with the colors the app uses and grab this from there?